### PR TITLE
Syntax issue on strings 92 & 108

### PR DESCRIPTION
**Pull request by kant**
on string 105: `all` or `call`?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,26 +102,26 @@ sess, _, err := provider_util.OpenProviderSession(conf, providerRegistry, conf.S
...
-Set the GOPATH as per your system and directory lets say `/home/user/test`   and create a sub directory under this by using following command
+Set the GOPATH as per your system and directory lets say `/home/user/test` and create a sub directory under this by using the following command
 
 ```
 $mkdir -p src/github.com/IBM
 ```
 
 ### Step 2
-Change the directory to `src/github.com/IBM`  nad check out the code
+Change the directory to `src/github.com/IBM` nad check out the code
```
Comment (narkarum) [inline README.md]: please replace nad with **and**

Comment (kant) [reply README.md]: done!

Comment (narkarum) [reply README.md]: still seeing same.

Comment (narkarum) [inline README.md]: Still seeing the same.

**Pull request merged**